xbmc/utils/HDRCapabilities.h

```diff
@@ -0,0 +1,32 @@
+/*
+ *  Copyright (C) 2005-2021 Team Kodi
+ *  This file is part of Kodi - https://kodi.tv
+ *
+ *  SPDX-License-Identifier: GPL-2.0-or-later
+ *  See LICENSES/README.md for more information.
+ */
+
+#pragma once
+
+class CHDRCapabilities
+{
+public:
+  CHDRCapabilities() = default;
+  ~CHDRCapabilities() = default;
+
+  bool SupportsHDR10() const { return m_hdr10; }
+  bool SupportsHLG() const { return m_hlg; }
+  bool SupportsHDR10Plus() const { return m_hdr10_plus; }
+  bool SupportsDolbyVision() const { return m_dolby_vision; }
+
+  void SetHDR10(bool supported) { m_hdr10 = supported; }
```

Comment:
Probably don't need the bool param for these methods as they will only ever be set to true.

Reply:
Indeed.